stock_location_orderpoint_source_relocate/tests/test_location_orderpoint.py

```diff
@@ -11,48 +11,110 @@
 
 
 class TestLocationOrderpoint(TestLocationOrderpointCommon, SourceRelocateCommon):
-    def test_auto_replenishment(self):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
         name = "Internal Replenishment"
-        replenishment_location = self.env["stock.location"].create(
+        cls.replenishment_location = cls.env["stock.location"].create(
             {
                 "name": name,
-                "location_id": self.wh.lot_stock_id.location_id.id,
+                "location_id": cls.wh.lot_stock_id.location_id.id,
             }
         )
-        internal_location = replenishment_location.create(
+        cls.internal_location = cls.env["stock.location"].create(
             {
                 "name": name,
-                "location_id": self.wh.lot_stock_id.id,
+                "location_id": cls.wh.lot_stock_id.id,
             }
         )
-        picking_type = self._create_picking_type(
-            name, replenishment_location, internal_location, self.wh
+        picking_type = cls._create_picking_type(
+            name, cls.replenishment_location, cls.internal_location, cls.wh
         )
-        route = self._create_route(
-            name, picking_type, replenishment_location, internal_location, self.wh
+        route = cls._create_route(
+            name,
+            picking_type,
+            cls.replenishment_location,
+            cls.internal_location,
+            cls.wh,
         )
+        cls.orderpoint = Form(cls.env["stock.location.orderpoint"])
+        cls.orderpoint.location_id = cls.internal_location
+        cls.orderpoint.route_id = route
+        cls.orderpoint = cls.orderpoint.save()
 
-        orderpoint = Form(self.env["stock.location.orderpoint"])
-        orderpoint.location_id = internal_location
-        orderpoint.route_id = route
-        orderpoint = orderpoint.save()
+        cls._create_incoming_move(10, cls.replenishment_location)
 
+    def test_auto_replenishment_without_relocation(self):
+        job_func = self.env["stock.location.orderpoint"].run_auto_replenishment
```

Comment:
```suggestion
        cls._create_incoming_move(10, cls.replenishment_location)
        cls.job_func = cls.env["stock.location.orderpoint"].run_auto_replenishment

    def test_auto_replenishment_without_relocation(self):
        job_func = self.job_func
```
Could you move this also to the setUpClass?

Reply:
done